src/video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -41,10 +41,11 @@ def analyze_video(video_path : str, time_range:int =None) -> Dict:
     Dict
         A dictionary of the facetuples found from the frames
     """
-
+	
```

Comment:
Nice tab ;)

Reply:
> why is it "fro" and not "from"

Because Jonas or Lukas made it that way originally, and no one must have complained in the pull request